017/main.go

```diff
@@ -10,10 +9,9 @@ import (
 func main() {
 	letters := []string{"J", "o", "e", "l", "\n"}
 	for {
-		if len(letters) == 0 {
-			break
+		if len(letters) > 0 {
+			fmt.Print(letters[0])
+			letters = letters[1:]
```

Comment:
Nothing is stopping the loop. It will print Joel but afterwards it will continue to run until it is manually killed